lib/y2snapper_common.pm

```diff
@@ -104,30 +104,16 @@ sub y2snapper_clean_and_quit {
     # Ensure yast2-snapper is not busy anymore
     wait_still_screen;
     # C'l'ose the snapper module
-    send_key "alt-l";
+    wait_screen_change { send_key "alt-l"; };
 
-    my $ret;
     if ($ncurses) {
-        $ret = wait_serial("yast2-snapper-status-0", 30);
+        wait_serial("yast2-snapper-status-0", 180);
     }
     else {
         # Wait until root gnome terminal is focussed, delete the directory and close window
-        $ret = check_screen('root-gnome-terminal');
+        assert_screen('root-gnome-terminal', timeout => 180);
     }
 
-    if (!$ret) {
-        $self->{mute_post_fail} = 1;
-        record_soft_failure 'bsc#1032831';
-        $self->y2snapper_failure_analysis;
```

Comment:
Is this method now called at all anymore? I guess we can also simplify the code further to put only the necessary parts into the post_fail_hook then

Reply:
just checked : 
1. yes you also can remove mute_post_fail variable because it was used only in part which you just deleted 
2. and we can't simplify the code because y2snapper_failure_analysis used in two places

please remove variable and I will merge PR

Reply:
I'm not sure what you mean by removing variable, as it's still used in yast2_snapper_ncurses.pm and yast2_snapper.pm. So I don't understand what should I remove.